mobile/examples/question_answering/android/prepare_model.py

```diff
@@ -0,0 +1,64 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+import transformers
+from pathlib import Path
+from onnxruntime.quantization import quantize_dynamic, QuantType
+from onnxruntime_extensions.tools import add_pre_post_processing_to_model as add_ppp
+from contextlib import contextmanager
+
+
+
+def get_model_from_huggingface(model_name: str = "csarron/mobilebert-uncased-squad-v2"):
+    """
+    Step 1. Download the model from huggingface and convert to onnx
+    """
+    tokenizer = transformers.AutoTokenizer.from_pretrained(model_name)
+    config = transformers.AutoConfig.from_pretrained(model_name)
+    model = transformers.MobileBertForQuestionAnswering.from_pretrained(model_name)
+    onnx_config = transformers.models.mobilebert.MobileBertOnnxConfig(config, "question-answering")
+
+    model_path = Path('app/src/main/res/raw/csarron_mobilebert_uncased_squad_v2.onnx')
+    onnx_inputs, onnx_outputs = transformers.onnx.export(tokenizer, model, onnx_config, 16, model_path)
+    return model_path
+
+
+def quantize_bert(model_path: Path):
```

Comment:
nit: bert might be a bit ambiguous, would it make sense to just say `quantize_model`?